patroni/postgresql/__init__.py

```diff
@@ -932,7 +938,28 @@ def get_history(self, timeline):
             except Exception:
                 logger.exception('Failed to read and parse %s', (history_path,))
 
-    def follow(self, member, role='replica', timeout=None, do_reload=False):
+    def follow(self,
+               member: Member,
+               role: Optional[str] = 'replica',
+               timeout: Optional[float] = None,
+               do_reload: Optional[bool] = False) -> Optional[bool]:
+        """Reconfigure postgres to follow a new member or use different recovery parameters.
+
+        Method may call `on_role_change` callback if role is changing.
+
+        :param member: The member to follow
+        :param role: The desired role, normally 'replica', but could also be a 'standby_leader'
+        :path The desired role, normally 'replica', but could also be 'standby_leader'
```

Comment:
what is `:path`?

Reply:
Nothing, seems to be accidental copy&paste, removed.